dell/xps/15-7590/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+  ];
+
+  # Set to true for just the first run, then disable it.
+  # boot.loader.efi.canTouchEfiVariables = lib.mkDefault true;
+
+  # Earlier font-size setup
+  console.earlySetup = true;
```

Comment:
I think I will later factor this out into an high DPI module.

Reply:
`console.earlySetup` is only valid in NixOS 20.03 onward; in 19.09 it's named `console.earlySetup`. Can we make it work for both versions, at least until NixOS 20.03 is stable?

Something like this should do it, I think:
```
  # ...
} // (if lib.versionAtLeast (lib.versions.majorMinor lib.version) "20.03"
      then { console.earlySetup = true; }
      else { boot.earlyVconsoleSetup = true; })
```

Reply:
@9999years yes. Just make a PR.